app/src/main/kotlin/com/simplemobiletools/gallery/helpers/MediaSideScroll.kt

```diff
@@ -0,0 +1,150 @@
+            }
+            MotionEvent.ACTION_MOVE -> {
+                val diffX = mTouchDownX - event.x
+                val diffY = mTouchDownY - event.y
+
+                if (Math.abs(diffY) > 20 && Math.abs(diffY) > Math.abs(diffX)) {
+                    var percent = ((diffY / ViewPagerActivity.screenHeight) * 100).toInt() * 3
+                    percent = Math.min(100, Math.max(-100, percent))
+
+                    if ((percent == 100 && event.y > mLastTouchY) || (percent == -100 && event.y < mLastTouchY)) {
+                        mTouchDownY = event.y
+                        mTouchDownVolume = getCurrentVolume()
+                    }
+
+                    volumePercentChanged(percent)
+                }
+                mLastTouchY = event.y
+            }
+            MotionEvent.ACTION_UP, MotionEvent.ACTION_CANCEL -> {
+                val diffX = Math.abs(event.x - mTouchDownX)
+                val diffY = Math.abs(event.y - mTouchDownY)
+                if (System.currentTimeMillis() - mTouchDownTime < CLICK_MAX_DURATION && diffX < 20 && diffY < 20) {
+                    onSpecialEvent()
+                }
+            }
+        }
+    }
+
+    fun handleBrightnessTouched(event: MotionEvent) {
+        when (event.action) {
+            MotionEvent.ACTION_DOWN -> {
+                mTouchDownX = event.x
+                mTouchDownY = event.y
+                mLastTouchY = event.y
+                mTouchDownTime = System.currentTimeMillis()
+                mSlideInfoText = "${context.getString(R.string.brightness)}:\n"
+                if (mTouchDownBrightness == -1)
+                    mTouchDownBrightness = getCurrentBrightness()
+            }
+            MotionEvent.ACTION_MOVE -> {
+                val diffX = mTouchDownX - event.x
+                val diffY = mTouchDownY - event.y
+
+                if (Math.abs(diffY) > 20 && Math.abs(diffY) > Math.abs(diffX)) {
+                    var percent = ((diffY / ViewPagerActivity.screenHeight) * 100).toInt() * 3
+                    percent = Math.min(100, Math.max(-100, percent))
+
+                    if ((percent == 100 && event.y > mLastTouchY) || (percent == -100 && event.y < mLastTouchY)) {
+                        mTouchDownY = event.y
+                        mTouchDownBrightness = mTempBrightness
+                    }
+
+                    brightnessPercentChanged(percent)
+                }
+                mLastTouchY = event.y
+            }
+            MotionEvent.ACTION_UP, MotionEvent.ACTION_CANCEL -> {
+                val diffX = Math.abs(event.x - mTouchDownX)
+                val diffY = Math.abs(event.y - mTouchDownY)
+                if (System.currentTimeMillis() - mTouchDownTime < CLICK_MAX_DURATION && diffX < 20 && diffY < 20) {
+                    onSpecialEvent()
+                }
+                mTouchDownBrightness = mTempBrightness
+            }
+        }
+        //mView.video_holder
```

Comment:
I commented it out, seems like this line is useless.

Reply:
just remove it altogether, seems like a mistake